### PR TITLE
Endurance 18 Initial Ingest Sheets

### PR DESCRIPTION
**Pull request by cwingard**
Initial sets of ingest CSVs (surface and profiler mooring) for the upcoming Endurance 18 deployments

### CODE REVIEW

File: CE09OSPM/CE09OSPM_D00018_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+# mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CE09OSPM/D00018/irid2shore/stc_status.*_*.txt,CE09OSPM-SBS01-00-STCENG000,telemetered,Pending,Incorrectly specified dataset and parser
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CE09OSPM/D00018/imm/mmp/E*.DAT,CE09OSPM-WFP01-00-WFPENG000,telemetered,Available,
```
Comment (s-pearce) [inline CE09OSPM/CE09OSPM_D00018_ingest.csv:3]: on the raw data server, using D00017 as an example data directory structure for OSPM, it looks like there should be a "cg_data" directory between "D00018" and the "imm" directory, unless I'm missing something.

**Review by s-pearce — CHANGES_REQUESTED** [review]
See my comment on the OSPM mooring directory structure.  It looks like in the raw data online there should be a "cg_data" directory name in the path.  
E.g. CE09OSPM/D00018/cg_data/imm/mmp/...

**Comment by cwingard** [comment]
Check again, there is no cg_data for the profiler moorings: https://rawdata.oceanobservatories.org/files/CE09OSPM/D00017/

**Review by s-pearce — APPROVED** [review]
looks good after all.

**Pull request merged**